gpt4all-bindings/cli/app.py

```diff
@@ -59,9 +59,14 @@ def repl(
         int,
         typer.Option("--n-threads", "-t", help="Number of threads to use for chatbot"),
     ] = None,
+    device: Annotated[
+        str,
+        typer.Option("--device", "-d", help="Device to use for chatbot - cpu, gpu or cuda"),
```

Comment:
The description should be clarified that these are *examples*, as you can also select specific devices this way by providing their full name:
```suggestion
        typer.Option("--device", "-d", help="Device to use for chatbot, e.g. gpu, amd, nvidia, intel. Defaults to CPU."),
```